docs/docs/05-reusable-components.md

```diff
@@ -239,6 +239,23 @@ Counter.propTypes = { initialCount: React.PropTypes.number };
 Counter.defaultProps = { initialCount: 0 };
 ```
 
+Alternatively, you can define `propTypes` and `defaultProps` as static properties of the class.
```

Comment:
I don't know the technical term, but yes these "static get" declarations work with just the Babel es2015 preset and nothing else.